packages/storybook/stories/va-checkbox-uswds.stories.tsx

```diff
@@ -139,7 +139,9 @@ const IndeterminateTemplate = ({}) => {
   };
 
   const handleSelectAllToggle = event => {
-    const checkboxes = document.querySelectorAll('.example-checkbox');
+    const checkboxes = document.querySelectorAll(
+      '.example-checkbox',
+    ) as unknown as HTMLInputElement[];
```

Comment:
To avoid the double cast and `unknown`, can this be refactored to something like: 

```
const checkboxes = Array.from(document.querySelectorAll<HTMLInputElement>('.example-checkbox'))
```

Reply:
This worked, thanks!